migrate/vault-pvc.yml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: data-vault-0
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: VOLUME_SIZE
```

Comment:
This is a total nitpick, not sure how we feel about newlines, but I like keeping the UI happy and adding one. 😂